db/migrate/20230816032232_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :user_id
```

Comment:
- [ ] As required for the project to correctly match the the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) give to you in the project requirements, kindly ensure that the foreign key to the `users` table is named `author_id`. I suggest you use the same approach, I mentioned in the previous comment even if it is added in a different file i.e. ``t.references :author, foreign_key: { to_table: 'users' }``

Reply:
They were already set up as foreign keys but I used a new migration to do it.
![image](https://github.com/PabloEGonz/blog-app/assets/68977631/73884b66-209f-4e69-94b4-348bcc11c315)